src/test/java/dk/sdu/se_f22/searchmodule/twowaysynonyms/TestUnitTwoWaySynonym.java

```diff
@@ -135,6 +135,39 @@ public void testUpdateGroupId(){
         assertEquals(operator.read(_defaultSynonym).groupId(), operator.read("PC").groupId());
     }
 
+    @Test
+    @DisplayName("Update group id of synonym")
+    public void testFilterMethod(){
+        // ARRANGE
+        ArrayList<String> synonymGroup1 = new ArrayList<>(){{
+            add(_defaultSynonym);
+            add("PC");
+        }};
+        ArrayList<String> synonymGroup2 = new ArrayList<>(){{
+            add("Socks");
+            add("tights");
+        }};
```

Comment:
Theres no need for creating two seperate lists here. Just 5 operator.create calls to generate the needed synonyms